src/graphql/query/teamsGoals.graphql

```diff
@@ -3,8 +3,16 @@ query GetGoals($teamId: Int, $limit: Int) {
 		totalCount
```

Comment:
Update this to use the new `goals` naming to get rid of this deprecation warning in GitHub.